examples/immutable-values/immutable_values.bal

```diff
@@ -0,0 +1,74 @@
+import ballerina/io;
+
+type Department record {
+    string name;
+    int id;
+};
+
+type Employee object {
+    string name;
+
+    function __init(string name) {
+        self.name = name;
+    }
+
+    function getName() returns string {
+        return self.name;
+    }
+};
+
+public function main() {
+    // define an `anydata` typed `map` with two entries
```

Comment:
This should be a valid sentence. Start with capital letter and end with a period. Check all the places.

Reply:
Fixed with 28377a1